packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -92,4 +94,10 @@ const defaultConfigs = {
   }
 };
 
+// Pull in the number of decimals for each identifier from the common getPrecisionForIdentifier. This is used within the
```

Comment:
@nicholaspai see here. I add the `priceFeedDecimals` from your default scaling from common for all price feeds. In the future, we will deprecate this when we move to all 18 decimal feeds but for now it keeps one single source of truth for all scaling between the bots and the voter dapp.

Reply:
I think we get this for free from our linter. If you try and make invalid JSON within this file husky will reject it. If you get it past husky then CI will reject it on the linter.

See the screenshot below which shows how husky detects this:
![image](https://user-images.githubusercontent.com/12886084/103651800-caa6f880-4f6a-11eb-9a1a-9ce106922988.png)

Reply:
Oh for real! Great